### PR TITLE
MSYS2: CI packaging fixes

### PR DESCRIPTION
**Pull request by dlmiles**
Run pacman -U --noconfirm mingw-*gtkwave*.pkg.tar.zst
  pacman -U --noconfirm mingw-*gtkwave*.pkg.tar.zst
  mv mingw-*gtkwave*.pkg.tar.zst ../../
  ../standalone_pkg.sh

../standalone_pkg.sh
  cp: cannot stat '/mingw64/bin/libffi-7.dll': No such file or directory

### CODE REVIEW

File: MSYS2/gtk3dll.inc [hunk 1]
```diff
@@ -1,3 +1,4 @@
+libcairo-gobject-2.dll
 libepoxy-0.dll
 libgdk-3-0.dll
 libgtk-3-0.dll
```
Comment (umarcor) [inline MSYS2/gtk3dll.inc:4]: Maybe add a newline at the end of this file.
It was missing already, not changed by this PR.

**Review by umarcor — APPROVED** [review]
LGTM!

File: MSYS2/gtk3dll.inc (hunk 1 above)
Comment (dlmiles) [reply MSYS2/gtk3dll.inc:4]: Understand that is my preference too (all txt file have EOL as last character sequence) but as a contributor I assume the project policy is as it is already, so it was left.

Comment (umarcor) [reply MSYS2/gtk3dll.inc:4]: Absolutely. Just pointed out because I saw it there. This PR is good to merge as-is.

**Comment by dlmiles** [comment]
Force push minor change to remove the entry, MSYS2/gtk3dll.inc libcairo-gobject-2.dll

Because it is already correctly present in MSYS/dll.inc and becomes duplicated/redundant in MSYS2/gtk3dll.inc from my original patch in this PR.

Like the previous version this also passes CI now.

**Pull request merged**